workflow/tests/base_results/results.csv

```diff
@@ -221,12 +221,9 @@ base-hvac-autosize-pthp-sizing-methodology-hers.xml,40.065,40.065,40.065,40.065,
+base-hvac-autosize-room-heat-pump-sizing-methodology-acca.xml,40.353,40.353,40.353,40.353,0.0,0.0,0.0,0.0,0.0,0.0,6.175,0.712,0.0,3.024,0.0,9.166,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.654,8.144,9.234,0.616,0.0,0.0,0.0,0.0,4472.4,2612.8,15.871,10.986,0.0,3.744,3.646,0.513,7.128,0.632,-2.014,0.0,0.0,6.648,-0.118,4.807,0.0,0.729,0.0,0.0,-11.389,0.0,0.075,-0.422,-0.046,2.936,-0.016,9.98,0.0,0.0,-5.631,-0.056,-1.121,-7.045,-0.158,0.0,0.0,9.901,1354.8,997.6,11399.6,2615.9,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+base-hvac-autosize-room-heat-pump-sizing-methodology-hers.xml,40.595,40.595,40.595,40.595,0.0,0.0,0.0,0.0,0.0,0.0,6.9,0.063,0.0,3.189,0.0,9.166,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.654,8.144,9.234,0.616,0.0,0.0,0.0,0.0,3805.9,2732.2,15.871,10.986,0.0,3.744,3.646,0.513,7.128,0.632,-2.014,0.0,0.0,6.648,-0.118,4.807,0.0,0.729,0.0,0.0,-11.389,0.0,0.075,-0.422,-0.046,2.936,-0.016,9.98,0.0,0.0,-5.631,-0.056,-1.121,-7.045,-0.158,0.0,0.0,9.901,1354.8,997.6,11399.6,2615.9,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+base-hvac-autosize-room-heat-pump-sizing-methodology-maxload.xml,41.134,41.134,41.134,41.134,0.0,0.0,0.0,0.0,0.0,0.0,7.369,0.0,0.0,3.323,0.0,9.166,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.654,8.144,9.234,0.616,0.0,0.0,0.0,0.0,3956.6,2833.2,15.871,10.986,0.0,3.744,3.646,0.513,7.128,0.632,-2.014,0.0,0.0,6.648,-0.118,4.807,0.0,0.729,0.0,0.0,-11.389,0.0,0.075,-0.422,-0.046,2.936,-0.016,9.98,0.0,0.0,-5.631,-0.056,-1.121,-7.045,-0.158,0.0,0.0,9.901,1354.8,997.6,11399.6,2615.9,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```

Comment:
Hmm, this is the opposite of the trend I would expect. MaxLoad sizing should have lower energy use than HERS, which should be lower than ACCA. That's how it works for the base-hvac-autosize-air-to-air-heat-pump files above, for example.